.github/workflows/ios-build.yml

```diff
@@ -85,4 +85,4 @@ jobs:
 
       - name: Build app
         working-directory: ${{ matrix.working-directory }}
-        run: npx react-native run-ios
+        run: npx react-native@latest run-ios --simulator="iPhone 14 Pro" --mode Debug --verbose --terminal /bin/zsh
```

Comment:
Why is this change necessary?

Reply:
Without `--terminal /bin/zsh` specified, the job fails at the beginning with message that no terminal has been specified 😮  `@latest` is recommended by Meta team iirc, and `--mode` and `--verbose` are not necessary but can be helpful with debugging most probably.